chef/cookbooks/neutron/recipes/cisco_support.rb

```diff
@@ -2,7 +2,7 @@
 
 neutron = node
 
-if neutron[:neutron][:networking_mode] == 'vlan'
+if neutron[:neutron][:ml2_type_drivers].include? 'vlan'
```

Comment:
Not that it hurts (on the contrary, I'd leave it), and more for my curiosity; aren't we enforcing through the validation the use of vlan when cisco is used?